src/index.js

```diff
@@ -0,0 +1,19 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import _ from 'lodash';
+import './style.css';
+import instagramBrands from './assets/instagram-brands.svg';
+
+function component() {
+  const element = document.createElement('div');
+  // Lodash, now imported by this script
+  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+  element.classList.add('hello');
+  return element;
+}
+
```

Comment:
Even this lodash sample script should be removed because we are more likely not to use it